src/core/version.py

```diff
@@ -0,0 +1 @@
+VERSION = "0.0.1"
```

Comment:
@vasylboliuk sinse this module doesn't do any specifig job, it is more common in python to name it something like __version__.py. You can look for open source packages on that approach

Reply:
`__version__`

Reply:
Done